pkgs/applications/editors/vscode/default.nix

```diff
@@ -32,15 +33,18 @@ in
       name = "code";
       exec = "code";
       icon = "code";
-      comment = "Visual Studio Code is a code editor redefined and optimized for building and debugging modern web and cloud applications";
+      comment = ''
+        Code editor redefined and optimized for building and debugging modern
+        web and cloud applications
+      '';
       desktopName = "Visual Studio Code";
       genericName = "Text Editor";
       categories = "GNOME;GTK;Utility;TextEditor;Development;";
     };
 
     buildInputs = if stdenv.system == "x86_64-darwin"
-      then [ unzip ]
-      else [ ];
+      then [ unzip makeWrapper libXScrnSaver ]
+      else [ makeWrapper libXScrnSaver ];
```

Comment:
@NeQuissimus I don't know very well nixexpr lang, could you show me how to do it outside ``if`` block.